_data/sidebars/resources.yml

```diff
@@ -0,0 +1,9 @@
+title: Resources
+title_url: /default
```

Comment:
Thanks, I went down the path of creating a default resource page and then backed out. It would be nice to have at some point but for now tutorials is probably ok. Fixed.